### ISSUE TITLE
Use an alternate download mirror for the bootstrapper

### ISSUE THREAD

**Issue by SCBionicle** [packwiz/packwiz-installer#18]
Github's free Rest API hits a limit and breaks the entire installer application when the auto updater can't reach the latest release.

Perhaps an alternate service could be used instead for the updater to pull from.

**Comment by comp500** [comment]
Yeah I'm planning to publish the installer on my own BunnyCDN repo, that won't have any ratelimits (and should be faster)
